lib/open_project/reporting/patches/open_project/configuration_patch.rb

```diff
@@ -24,7 +24,23 @@ module OpenProject::Reporting::Patches
   module OpenProject::ConfigurationPatch
     def self.included(base)
       base.class_eval do
+        extend ModuleMethods
+
         @defaults['cost_reporting_cache_filter_classes'] = true
+
+        if config_loaded_before_patch?
+          @config['cost_reporting_cache_filter_classes'] = true
+        end
+      end
+    end
+
+    module ModuleMethods
+      def config_loaded_before_patch?
+        @config.present? && !@config.has_key?('cost_reporting_cache_filter_classes')
```

Comment:
I don't think that this will ever be a problem, but keep in mind that:

````
[25] pry(main)> {}.present?
=> false
````

I don't believe there will every be a present configuration that is empty, but then: What do I know? This is why this `present?` leaves a bad taste...